javascripts/discourse/initializers/initialize-composer-toggle-hijack.js

```diff
@@ -1,53 +1,240 @@
+          @discourseComputed('seq')
+          content() {
+            let items = []
+
+            if (
+              this.action === REPLY &&
+              this.topic &&
+              this.topic.isPrivateMessage &&
+              this.topic.details &&
+              (this.topic.details.allowed_users.length > 1 ||
+                this.topic.details.allowed_groups.length > 0) &&
+              !this.isEditing &&
+              _topicSnapshot
+            ) {
+              items.push({
+                name: I18n.t(
+                  'composer.composer_actions.reply_as_new_group_message.label'
+                ),
+                description: I18n.t(
+                  'composer.composer_actions.reply_as_new_group_message.desc'
+                ),
+                icon: 'plus',
+                id: 'reply_as_new_group_message',
+              })
+            }
+
+            if (
+              this.action !== CREATE_TOPIC &&
+              this.action !== CREATE_SHARED_DRAFT &&
+              this.action === REPLY &&
+              this.topic &&
+              !this.topic.isPrivateMessage &&
+              !this.isEditing &&
+              _topicSnapshot
+            ) {
+              items.push({
+                name: I18n.t(
+                  'composer.composer_actions.reply_as_new_topic.label'
+                ),
+                description: I18n.t(
+                  'composer.composer_actions.reply_as_new_topic.desc'
+                ),
+                icon: 'plus',
+                id: 'reply_as_new_topic',
+              })
+            }
+
+            if (
+              (this.action !== REPLY && _postSnapshot) ||
+              (this.action === REPLY &&
+                _postSnapshot &&
+                !(this.replyOptions.userAvatar && this.replyOptions.userLink))
+            ) {
+              items.push({
+                name: I18n.t('composer.composer_actions.reply_to_post.label', {
+                  postUsername: _postSnapshot.username,
+                }),
+                description: I18n.t(
+                  'composer.composer_actions.reply_to_post.desc'
+                ),
+                icon: 'share',
+                id: 'reply_to_post',
+              })
             }
 
-            options.action = "privateMessage";
-            options.recipients = usernames;
-            options.archetypeId = "private_message";
-            options.skipDraftCheck = true;
+            if (
+              this.siteSettings.enable_personal_messages &&
+              this.action !== PRIVATE_MESSAGE &&
+              !this.isEditing
+            ) {
+              items.push({
+                name: I18n.t(
+                  themePrefix(
+                    'custom_composer_actions.reply_as_private_message.label'
+                  )
+                ),
+                description: I18n.t(
+                  themePrefix(
+                    'custom_composer_actions.reply_as_private_message.desc'
+                  )
```

Comment:
These locales seem to be missing

<img width="319" alt="Screen Shot 2022-09-13 at 2 35 13 PM" src="https://user-images.githubusercontent.com/30090424/190013577-abc7fca2-4d3d-47d3-9681-433ba84db972.png">